docs/source/tutorials/multiannotator.ipynb

```diff
@@ -0,0 +1,555 @@
+    "\n",
+    "    m = len(means)  # number of classes\n",
+    "    n = sum(sizes)\n",
+    "    local_data = []\n",
+    "    labels = []\n",
+    "\n",
+    "    for idx in range(m):\n",
+    "        local_data.append(\n",
+    "            np.random.multivariate_normal(mean=means[idx], cov=covs[idx], size=sizes[idx])\n",
+    "        )\n",
+    "        labels.append(np.array([idx for i in range(sizes[idx])]))\n",
+    "    X_train = np.vstack(local_data)\n",
+    "    true_labels_train = np.hstack(labels)\n",
+    "\n",
+    "    # Compute p(true_label=k)\n",
+    "    py = np.bincount(true_labels_train) / float(len(true_labels_train))\n",
+    "    \n",
+    "    noise_matrix_better = generate_noise_matrix_from_trace(\n",
+    "        m,\n",
+    "        trace=0.9 * m,\n",
+    "        py=py,\n",
+    "        valid_noise_matrix=True,\n",
+    "        seed=seed,\n",
+    "    )\n",
+    "    \n",
+    "    noise_matrix_worse = generate_noise_matrix_from_trace(\n",
+    "        m,\n",
+    "        trace=0.5 * m,\n",
+    "        py=py,\n",
+    "        valid_noise_matrix=True,\n",
+    "        seed=seed,\n",
+    "    )\n",
+    "\n",
+    "    # Generate our noisy labels using the noise_matrix for specified number of annotators.\n",
+    "    s = pd.DataFrame(\n",
+    "        np.vstack(\n",
+    "            [\n",
+    "                generate_noisy_labels(true_labels_train, noise_matrix_better)\n",
+    "                if i < num_annotators - 5\n",
+    "                else generate_noisy_labels(true_labels_train, noise_matrix_worse)\n",
+    "                for i in range(num_annotators)\n",
+    "            ]\n",
+    "        ).transpose()\n",
+    "    )\n",
+    "\n",
+    "    # Each annotator only labels approximately 10% of the dataset\n",
+    "    # (unlabeled points represented with NaN)\n",
+    "    s = s.apply(lambda x: x.mask(np.random.random(n) < 0.9))\n",
+    "    s.dropna(axis=1, how=\"all\", inplace=True)\n",
+    "\n",
+    "    row_NA_check = pd.notna(s).any(axis=1)\n",
+    "\n",
+    "    return {\n",
+    "        \"X_train\": X_train[row_NA_check],\n",
+    "        \"true_labels_train\": true_labels_train[row_NA_check],\n",
+    "        \"multiannotator_labels\": s[row_NA_check].reset_index(drop=True),\n",
+    "    }\n",
+    "\n",
+    "data_dict = make_data()\n",
+    "\n",
+    "X = data_dict[\"X_train\"]\n",
+    "multiannotator_labels = data_dict[\"multiannotator_labels\"]\n",
+    "true_labels = data_dict[\"true_labels_train\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e95d31b2",
+   "metadata": {},
+   "source": [
+    "For this tutorial we are using a toy dataset that has 50 annotators and 160 examples. There are three possible classes, `0`, `1` and `2`.\n",
+    "\n",
+    "Each annotator annotates approximately 10% of the examples. We also synthetically made the last 5 annotators in our toy dataset have much noisier labels than the rest of the annotators."
```

Comment:
Add final cell here that prints a `head()` of the data that the user would have in practice (not the "true"-stuff).  We want to give reader a sense of what their data should look like. 

If we've designed our APIs right, the only work  the user needs to do is get their data formatted properly, so it's imperative we make it as clear as possible what the exact format should look like

Reply:
When I run this code, the `multiannotator_labels` object contains floats as the labels, but I'd presume it would make more sense for these to be ints?

Reply:
Basically you'll want to envision users trying to run our functions on every single malformatted dataset that could possibly exist :)  

And we want to preclude these issues from happening via a clear tutorial and docstrings!

Reply:
Yes all labels are saved as dytpe float to allow for NaN values (which numpy defines as float), added note to clarify

Reply:
Actually since we are creating the dataset and now that we want it to have annotator IDs, it has to be pandas. And I think you can in fact make pandas only have integer values with missing entries?

https://pandas.pydata.org/docs/user_guide/integer_na.html#:~:text=In%20Working%20with%20missing%20data,this%20may%20not%20matter%20much.

Reply:
Changed to used pd.NA types